src/style.css

```diff
@@ -1 +1,81 @@
 /* add styles here */
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300px;
+  line-height: 21.86px;
+  text-align: left;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(/src/fonts/Avenir-Book.ttf) format(ttf);
+  src: url(/src/fonts/Avenir-Heavy.ttf) format(ttf);
+  src: url(/src/fonts/Avenir.ttc) format(ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+
+body {
+  margin: 0;
+  font-family: Avenir, sans-serif;
+  padding: 0;
+  width: 100vw; /* Полная ширина окна */
+  height: 100vh; /* Полная высота окна */
+  box-sizing: border-box;
+}
+
+.search-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%; /* Растягиваем контейнер на всю ширину страницы */
+  padding: 0; /* Отступы по краям */
+  box-sizing: border-box; /* Включаем отступы в ширину */
+}
+
+/* Стиль для первой (большой) строки поиска */
+
+.search-field.large input {
+  border: none;
+  padding: 20px 20px 20px 50px; /* Увеличенные отступы для иконки */
+  background-color: white;
+  background-image: url('/src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 15px center;
+  background-size: 19px 19px;
+  width: 100%;
+  height: 70px; /* Высота для большой строки поиска */
+  box-sizing: border-box;
+  font-size: 18px; /* Размер текста побольше */
+  margin-bottom: 20px; /* Отступ между строками */
```

Comment:
The `margin-bottom` property is being set on the `input` element. According to the checklist, it's recommended to use either `margin-top` or `margin-bottom` but not both to avoid potential margin collapse issues.